exercises/hello-world/example.ml

```diff
@@ -0,0 +1,4 @@
+let greet subject =
+  match subject with
+  | None    -> "Hello, World!"
+  | Some(s) -> String.concat "" ["Hello, "; s; "!"]
```

Comment:
Great! I searched for a concatenation operator, but I missed it. I will change the implementation.